_includes/cloudcode/cloud-code.md

```diff
@@ -24,6 +24,8 @@ Parse.Cloud.define("averageStars", async (request) => {
     sum += results[i].get("stars");
   }
   return sum / results.length;
+},{
+  fields:['movie']
```

Comment:
I figured it might be an easy way to warm into the cloud validation. Considering `request.params.movie` is used in the query, I thought it would make sense to make sure it's not `null`.

Reply:
I think it's confusing as it isn't explained so if someone doesn't look at the cloud validation section they may not have any idea what it does.